proxy.go

```diff
@@ -35,6 +35,7 @@ type ProxyHttpServer struct {
 	harLog            *har.Har
 	harLogEntryCh     chan harReqAndResp
 	harFlushRequest   chan string
+	HarFlushToMemory  chan chan *har.Har
```

Comment:
a `chan` or `chan *har.Har` ? what are you trying to do here ? would you like to set the HAR logged in a mode in which it would sent all the requests through a channel you control ?  I see why we would like to do that.. so we can have a goroutine that handles incoming requests for full analysis (although the proxy can already do it).. but I can see you want it in analyzed HAR format...

what I don't see is how this implementation would do what I just explained..